bodhi/client/__init__.py

```diff
@@ -80,7 +80,8 @@ def new(user, password, **kwargs):
         except bindings.BodhiClientException as e:
             click.echo(str(e))
         except Exception as e:
-            traceback.print_exc()
```

Comment:
I think we should keep this except block as it is since it's a general (unexpected) exception and that means it's a bug.